File: 2078-two-furthest-houses-with-different-colors/2078-two-furthest-houses-with-different-colors.java
class Solution {
    public int maxDistance(int[] colors) {
        int dist =0;
        for(int i=0; i < colors.length; i++){
            for(int j=i; j < colors.length; j++)
            if(colors[i] == colors[j]){
                continue;
            }else{
                dist = Math.max(dist, Math.abs(i - j));
            }
        }
        return dist;
    }
}

File: 2078-two-furthest-houses-with-different-colors/2078-two-furthest-houses-with-different-colors.cpp
class Solution {
public:
    int maxDistance(vector<int>& colors) {
        int d =0;
        for(int i=0; i < colors.size(); i++){
            for(int j=i; j < colors.size(); j++)
            if(colors[i] == colors[j]){
                continue;
            }else{
                d = max(d, abs(i - j));
            }
        }
        return d;
    }
};